code/modules/mob/living/living.dm

```diff
@@ -24,6 +24,7 @@
 /mob/living/proc/prepare_data_huds()
 	med_hud_set_health()
 	med_hud_set_status()
+	update_dnr_hud() // SKYRAT EDIT ADDITION - DNR HUD
```

Comment:
Does it really need to be here, instead of having it modular? It's not like if extending this proc was really that hard to do modularly.

Reply:
oh yeah it can be done modularly im just dumb